#!/usr/bin/env python
PACKAGE = "tarkbot_driver_yolo"
 
from dynamic_reconfigure.parameter_generator_catkin import *
 
gen = ParameterGenerator()

gen.add("outer_ring_center_target", double_t, 0, "outer_ring_center_target", 0.26,  0, 1)
gen.add("outer_ring_vel_z_P", double_t, 0, "outer_ring_vel_z_P", 0.01,  0, 0.02)
gen.add("outer_ring_vel_z_D", double_t, 0, "outer_ring_vel_z_D", 0.001,  0, 0.01)

gen.add("Inner_ring_center_target", double_t, 0, "Inner_ring_center_target", 0.75,  0, 1)
gen.add("Inner_ring_vel_y_P", double_t, 0, "Inner_ring_vel_y_P", 0.001,  0, 0.02)
gen.add("Inner_ring_vel_y_D", double_t, 0, "Inner_ring_vel_y_D", 0.003,  0, 0.01)
gen.add("Inner_ring_vel_z_P", double_t, 0, "Inner_ring_vel_z_P", 0.006,  0, 0.02)
gen.add("Inner_ring_vel_z_D", double_t, 0, "Inner_ring_vel_z_D", 0.001,  0, 0.01)

exit(gen.generate(PACKAGE, "tarkbot_driver_yolo", "params"))  
